schema/gendummydata.py

```diff
@@ -41,8 +53,21 @@
 RANDOM_TLDS = ("edu", "com", "org", "net", "tw", "ru", "pl", "de", "es")
 RANDOM_URL = ("http://www.", "ftp://ftp.", "http://", "ftp://")
 RANDOM_LOCS = ("pub", "release", "files", "downloads", "src")
-FORTUNE_FILE = os.environ.get("FORTUNE_FILE", "/usr/share/fortune/cookie")
 
+if (os.name == 'posix'):
+    OS = getosname()
+    if OS == 'arch':
+        FORTUNE_FILE = "/usr/share/fortune/cookie"
+    elif OS == 'debian':
+        FORTUNE_FILE = "/usr/share/games/fortunes/fortunes"
+    elif OS == 'ubuntu':
+        # Needs to be verified.
+        FORTUNE_FILE = "/usr/share/games/fortunes/fortunes"
+    else:
+        print(f"Please check and Install Fortune for your os: {sys.platform} and set path for FORTUNE_FILE env var.")
```

Comment:
Both words in `Install Fortune` shouldn't need to be capitalized, right?

We can remove this since we're only support Arch and Debian/Ubuntu, which would be covered in the above two `if/elif` blocks anyway, right?

Reply:
I donot think thats much of an issue, should make F small though because someone might just try to `sudo apt install Fortune` and complain XD